hack/testdata/pod-with-precision.json

```diff
@@ -1,24 +1,24 @@
 {
-    "apiVersion": "v1",
-    "kind": "Pod",
-    "metadata": {
-        "name": "pod-with-precision"
-    },
-    "spec": {
-      "activeDeadlineSeconds": 2147483647,
-      "containers": [
-        {
-          "name": "kubernetes-pause",
-          "image": "k8s.gcr.io/pause:3.1"
-        }
-      ],
-      "restartPolicy": "Never",
-      "securityContext": {
-          "supplementalGroups": [
-              0,
-              1000030003,
-              2147483647
-          ]
+  "apiVersion": "v1",
+  "kind": "Pod",
+  "metadata": {
+    "name": "pod-with-precision"
+  },
+  "spec": {
+    "activeDeadlineSeconds": 2147483647,
+    "containers": [
+      {
+        "name": "kubernetes-pause",
+        "image": "k8s.gcr.io/pause:3.2"
       }
+    ],
+    "restartPolicy": "Never",
+    "securityContext": {
+      "supplementalGroups": [
+        0,
+        1000030003,
+        2147483647
+      ]
     }
-}
+  }
+}
```

Comment:
lint: TerminatingNewline

Reply:
This is auto formatted after a find and replace
I'm not sure how useful a trailing newline is in a JSON data file.